docz/stake.mdx

```diff
@@ -0,0 +1,92 @@
+---
+name: Stake & UnStake
+route: /staking
+menu: SDK
+---
+
+import { Props } from 'docz';
+import { stakeTransaction } from './transaction'
+
+# Staking
+
+---
+### 1. Initialize the library
+
+```ts
+import { SolidoSDK } from '@lidofinance/solido-sdk';
+import { Connection } from '@solana/web3.js';
+
+// Change rpc endpoint to yours, or whole connection
+const connection = new Connection('https://test.rpcpool.com');
+
+const solidoSDK = new SolidoSDK('mainnet-beta', connection, 'your_solana_referral_address');
+```
+
+### 2. Prepare stake transaction
+
+<Props of={stakeTransaction} />
+
+```ts
+try {
+  const stakeTransaction = await solidoSDK.getStakeTransaction({
+    amount: 20,
+    payerAddress: new PublicKey('user_address'),
+  });
+} catch (e) {
+  // Handle Errors
+}
+```
+
+#### ❗️ Note
+
+You may get exception if amount exceed maximum available SOL-s in balance. In order to prevent this exception, you can call
+```ts
+const maxStakeAmount = await solidoSDK.calculateMaxStakeAmount(new PublicKey('user_address'));
+```
+
+### 3. Sign and confirm transaction
+
+Here you can call ready function from SDK:
+
+```ts
+try {
+  // use transactionHash for generating links to explorers
+  const transactionHash = await solidoSDK.stake({
+    transaction: stakeTransaction,
+    wallet: WalletInstance,
+    setTxStage: setTxStageCallback,
+  });
+} catch (e) {
+  // Handle Errors
+}
+```
+
+Or, call sign & confirm methods yourself for total control in transaction stages:
```

Comment:
`Or call sign & confirm methods by yourself for total control transaction stages`